src/main/kotlin/org/rucca/cheese/model/PatchTaskMembershipRequestDTO.kt

```
@@ -0,0 +1,17 @@
+/**
+ * @param deadline
+ * @param approved
+ */
+data class PatchTaskMembershipRequestDTO(
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("deadline")
+    val deadline: kotlin.Long? = null,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("approved")
+    val approved: kotlin.Boolean? = null
+) {}
```

detekt

[warning] The class or object PatchTaskMembershipRequestDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)